### ISSUE TITLE
CORE-AAM 1.1: Ensure default mappings for aria-valuemin and aria-valuemax are properly mapped for all associated roles

### ISSUE THREAD

**Issue by richschwer** [w3c/aria#463]
Moved from tracker action: https://www.w3.org/WAI/ARIA/track/actions/2119

**Comment by cookiecrook** [comment]
Rich's comment from Tracker:

Authors MUST set the aria-valuemin, aria-valuemax, and aria-valuenow attributes. If missing, their implicit values follow the same rules as the HTML range input type:

If aria-valuemin is missing or not a number, it defaults to 0 (zero).
If aria-valuemax is missing or not a number, it defaults to 100.
If aria-valuenow is missing or not a number, it defaults to the value half way between aria-valuemin and aria-valuemax.
If aria-valuenow is present but less than aria-valuemin, it defaults to the value of aria-valuemin.
If aria-valuenow is present but greater than aria-valuemax, it defaults to the value of aria-valuemax.

**Comment by cookiecrook** [comment]
Joseph's comment from Tracker:

Regarding Rich's comment 19 Oct 2016, the rules he cites are for slider, which matches the HTML range input type. They also apply to the scrollbar role.

It's different for spinbutton (HTML number input type):
- if aria-valuemax is missing, then the spinbutton has no upper bound (Accessibility API dependent).
- if aria-valuemin is missing, the spinbutton has no lower bound (Accessibility API dependent).
- a missing aria-valuenow defaults to zero.

Also, all of these mappings are based on the ARIA 1.1 spec text for slider, scrollbar, and spinbutton.

**Comment by Joanmarie** [comment]
Silly question: Isn't that what I already committed for WebKit? https://trac.webkit.org/changeset/208739

**Comment by cookiecrook** [comment]
> Silly question: 

How is that a silly question?

> Isn't that what I already committed for WebKit? https://trac.webkit.org/changeset/208739

Great. Closing issue. Thanks Joanie.
